demo/app/src/context/appContext.js

```diff
@@ -5,33 +5,39 @@ const AppContext = createContext();
 
 export default function AppProvider({ children }) {
   const today = new Date();
-  const [geoJson, setGeoJson] = useState();
-  const [bbox, setBbox] = useState();
-  const [dateRange, setDateRange] = useState([
+  /*
+    userParams interface:
+    {
+      bbox: tuple<number>,
+      dateRange: tuple<Date>
+    }
+  */
+  const [userParams, setUserParams] = useState({
```

Comment:
suggesting jsdoc here

```suggestion
  /**
   * @typedef {object} UserParams
   * @property {[Date, Date]} dateRange
   * @property {number[]} bbox
  */
  const [
    /** @type {UserParams} */ userParams,
    setUserParams
  ] = useState({
```